docs/REST-API-Reference.md

```diff
@@ -9,26 +9,14 @@ subject to change at any time. Feedback through our GitHub issues is appreciated
 - [Introduction](#introduction)
 - [Resource Object Schemas](#resource-object-schemas)
 - [API Endpoints](#api-endpoints)
-  - [Bundle Actions API](#bundle-actions-api)
```

Comment:
Why is this being removed?

Reply:
It's not by me. Running the pre-commit.sh did.

Reply:
Our pre-commit script is deleting the documentation. That doesn't seem right.